apps/devhub/tasks.py

```diff
@@ -243,10 +243,10 @@ def skip_signing_warning(result):
 
 @task(soft_time_limit=settings.VALIDATOR_TIMEOUT)
 @write
-def compatibility_check(upload_id, app_guid, appversion_str, **kw):
+def compatibility_check(upload_pk, app_guid, appversion_str, **kw):
     log.info('COMPAT CHECK for upload %s / app %s version %s'
-             % (upload_id, app_guid, appversion_str))
-    upload = FileUpload.objects.get(pk=upload_id)
+             % (upload_pk, app_guid, appversion_str))
```

Comment:
Only log statement I'm keeping with a pk, because it's made before fetching the `FileUpload` instance, and therefore we don't have the `uuid` yet. I could move it below but I think it's useful to have it before doing anything.